README.md

```diff
@@ -29,3 +29,35 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons? What is it in terms of the number of flips? Add your
 answer to this markdown file.
 
+## My Runtime Analysis
+
+When pancakeSort() is called, it enters a for loop that will iterate through
+ the array backwards. This operation will be a complexity of $\Theta(n)$. 
+
+Inside this loop is the function findMaxIndex() which looks through a sub-array 
+of the array given. Since it's not the entire length of $n$ every call (though 
+the worst case would be $n$), I will use $m$. The time complexity of 
+findMaxIndex() is $\Theta(m)$.
```

Comment:
What is $m$?

Reply:
$m$ is the linear amount of time that it takes the algorithm to run a **subset** of the array. The algorithm will not always run a full number of n times through, for example, the flip() function. A subset is passed to function, therefore the asymptotic complexity uses $m$ to represent this linear complexity subset.

Reply:
And what is $m$ bounded by?

Reply:
$m$ is just another linear time complexity, but different from $n$. In my analysis, it's similar to saying $\Theta(n^2)$, however, since certain functions like flip() and findMaxIndex() will not use the full array with length $n$ every iteration, I make the distinction that $m$ is a subset of $n$. Therefore leaving the final complexity at $\Theta(n*m)$

Reply:
That sounds weird though -- there's only one input size (the number of pancakes). Would you not want to know the complexity in terms of that?

Reply:
Would it be preferred if I left things in terms of n (e.i. $\Theta(n^2)$? It just doesn't feel right to call it $n^2$ because not every variation of those helper functions will have the full length of $n$.

Reply:
Well, what matters is how the amount of work grows as the input size grows. So even if the actual amount of work is different, as long as it grows at the same rate, it doesn't make a difference.